pwnlib/ui.py

```diff
@@ -97,11 +97,11 @@ def yesno(prompt, default=None):
         cur = default
         while True:
             k = term.key.get()
-            if   k in ('y', 'Y', '<left>') and cur != True:
+            if   k in ('y', 'Y', '<left>') and cur is not True:
```

Comment:
This should likely be `not cur` or similar.  We should generally not check for `is True` or `is False`.

Reply:
Ping @Arusekk

Reply:
Here the value can be either of three: `True`, `False` or `None`. You are probably right, if you prefer so, you can commit this further change.